File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        cleaned = "".join(c.lower() for c in s if c.isalnum())
        if cleaned.lower() == cleaned[::-1].lower():
            return True
        else:
            return False

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
#include<string>
class Solution {
public:
    bool isPalindrome(string s) {
        string result = "";
        int i, j;
        for (char char1 : s){
            if (isalnum(char1)){
                result += tolower(char1);
            }
            
        }
        for (i = 0, j = result.size() - 1 ; i < j, j > 0; i++, j-- ){
            if (result[i]!=result[j]){
                return false;
            }
        }
    return true;
    }
};